packages/flutter_tools/test/general.shard/channel_test.dart

```diff
@@ -46,6 +46,41 @@ void main() {
       await simpleChannelTest(<String>['channel', '-v']);
     });
 
+    testUsingContext('sorted by stability', () async {
```

Comment:
I would also include one where a channel is missing or extra channels are added

Reply:
Done, Added both of these in the same test.
For when a channel is missing, I couldn't use containsAllInOrder Matcher so I used a different method